android_webview/BUILD.gn

```diff
@@ -447,8 +447,10 @@ if (defined(android_secondary_abi_toolchain)) {
     }
   }
 } else {
-  # 32-bit WebView library used in 32-bit only builds of 64/32 targets.
-  webview_alternate_library("monochrome_64") {
+  if (!android_64bit_target_cpu) {
+    # 32-bit WebView library used in 32-bit only builds of 64/32 targets.
+    webview_alternate_library("monochrome_64") {
+    }
   }
```

Comment:
2nd review:
这个我建议改成下面这样：
```python
if (android_64bit_target_cpu) {
  if (defined(android_secondary_abi_toolchain)) {
    # Note here that on a given system, the webview-only library needs the same
    # library name as the browser library, since the system webview factory will
    # differentiate only by ABI.
    if (current_toolchain == android_secondary_abi_toolchain) {
      # This target is the 32-bit WebView library that pairs with a 64-bit
      # browser. It is suffixed with _64 because its name must match the 64-bit
      # browser library.
      webview_alternate_library("monochrome_64") {
      }
    } else {
      # Inverse of above, for the original 32-bit case.
      webview_alternate_library("monochrome") {
      }
    }
  }
} else {
  # 32-bit WebView library used in 32-bit only builds of 64/32 targets.
  webview_alternate_library("monochrome_64") {
  }
}
```

核心逻辑抽象如下：
```python
if (android_64bit_target_cpu) {
  if (defined(android_secondary_abi_toolchain)) {
    处理 arm64/x86_64
  }
  处理 riscv64, 当前没有 2nd abi，所以什么也不做
} else {
  处理 arm and x86
}
```
你现在这样改的改动比较小，但是感觉在原来处理 arm和 x86 的分支里突然插入 64 位的逻辑很突兀，以后也不好理解